cmd/create.go

```diff
@@ -92,6 +95,16 @@ var createCmd = &cobra.Command{
 			}
 		}
 
+		if flagPath == "" {
+			tprogram = tea.NewProgram(textinput.InitialTextInputModel(options.ProjectPath, "Where should the project be created? (use '.' for current directory)", project))
```

Comment:
TBH Idk if i like forcing a user to add a path to explicitly state where the project should be created.
Also, where would the option be in the CLI?